common/list/lists-mws-en.html

```diff
@@ -0,0 +1,241 @@
+---
+{
+	"title": "Lists for MWS",
+	"language": "en",
+	"altLangPage": "lists-mws-fr.html",
+	"dateModified": "2024-03-27"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<p>To accomodate MWS product limitations the lists css classes are applied on parent element</p>
```

Comment:
This is fine, but expand the abbreviation "MWS" and remove "limitations" which you can replace by "regarding". 

Changing the verb explaining why this is applied on the parent element.
```suggestion
<p>To accommodate managed web service product regarding the lists CSS classes which must be applied on parent element.</p>
```